doc/metric_spec.md

```diff
@@ -0,0 +1,111 @@
+# Metrics specification:
+
+This is a specification of metrics exposed by the ha_cluster exporter.
+
+All metrics from the exporter start with the prefix `ha_cluster`
+
+Below you have a complete specification, ordered by component.
+
+1. [pacemaker](#pacemaker)
+2. [drbd](#drbd)
+3. [sbd](#sbd)
+4. [corosyncl](#corosync)
+
+# Pacemaker 
+
+The Pacemaker cluster metrics are atomic metrics and represent and updated snapshot of the HA cluster, retrieved fetching the XML CIB of pacemaker.
+
+Some of the pacemaker metrics like `ha_cluster_node_resources` and `ha_cluster_nodes` metrics with labels share a common trait:
+ 
+they can be either set to `1` or they are absent, this is because they track the real state of the cluster resources monitored.
+
+1. [ha_cluster_node_resources](#ha_cluster_node_resources)
+2. [ha_cluster_nodes](#ha_cluster_nodes)
+3. [ha_cluster_nodes_configured_total](#ha_cluster_nodes_configured_total)
+4. [ha_cluster_resources_configured_total](#ha_cluster_resources_configured_total)
+
+
+
+## ha_cluster_node_resources
+
+This metric show the current status of  a node resource. 
+
+A resource that previously was in the cluster but isn't anymore, will not monitored. Example:
+
+```ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="cluster_md",role="started",status="active"} 1```
+
+The metric will absent and not `0`
+
+
+All the values are 1:1 with Pacemaker schema.
+
+- `managed`: indicates `true` or `false` if the resource is managed in cluster
+- `node_name`: name of node of cluster monitored
```

Comment:
Does it mean the node where the resource status is retrieved from or the node where the resource is running?

I'm asking because for clone or promotable clone (master/slave type), there are usually multiple instances of a primitive resource running in the cluster.

Reply:
nope we use crm_mon. I will remove the monitored word

Reply:
But that doesn't answer my question :-)
Is "node_name" the node where the resource is running on, or where crm_mon is running on?